crates/sui-protocol-config/src/lib.rs

```diff
@@ -971,6 +976,13 @@ impl ProtocolConfig {
                 cfg.gas_model_version = Some(3);
                 cfg
             }
+            5 => {
+                let mut cfg = Self::get_for_version_impl(version - 1);
+                // TODO(devx): Bump this in next PR
+                cfg.gas_cost_table_version = Some(1);
+                cfg.gas_model_version = Some(4);
```

Comment:
What's the relationship between gas_cost_table_version and gas_model_version? Do you really need gas_cost_table_version as an individual number?

Reply:
Good question -- we discussed offline and it would be better if `gas_cost_table` were not a separate number otherwise it could bring up weird compatibility issues at a later date possibly. I'll remove it and we'll only have `gas_model_version`.